app/tests/test_amity.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from app.cp.amity import Amity
+
+
+class Test_amity_class(unittest.TestCase):
+
+    def setUp(self):
+        self.amity = Amity()
+
+    def tearDown(self):
+        del self.amity
+
+    """
+    Tests for the create_room function in class Amity.
+    The tests validate input and output data and also see if
+    specified constraints are met and EdgeCases are handled. create_room
+    function takes the room_name and Room_Type as arguments.
+    """
+
+    def test_creating_room_with_wrong_room_type(self):
+        self.assertEqual(self.amity.create_room("Narnia", "Quiet_Room"),
+                         "Room_Type can only be OFFICE or LIVING_SPACE",
+                         msg="Room_Type can only be OFFICE or LIVING_SPACE")
+
+    def test_created_rooms_persisted(self):
+        self.amity.create_room("Hogwarts", "Office")
+        self.assertListEqual([{'OFFICE': 'Hogwarts'}], self.amity.all_rooms)
+
+    def test_successful_office_creation(self):
+        self.assertEqual(self.amity.create_room("Hogwarts", "Office"),
+                         "OFFICE successfully created!")
+
+    def test_successful_living_space_creation(self):
+        self.assertTrue(self.amity.create_room("Barmuda", "L") ==
+                        "LIVING_SPACE successfully created!")
+
+    def test_creation_of_multiple_rooms_at_once(self):
+        self.amity.create_room("Hogwarts Mombasa", "Office")
+        self.assertListEqual(self.amity.offices, ['Hogwarts', 'Mombasa'])
+
+    def test_living_space_isnt_created_when_office_is_specified(self):
+        self.assertNotEqual("LIVING_SPACE successfully created!",
```

Comment:
Instead of testing for negative, test for the actual message expected for the condtion